src/battle-dex.ts

```diff
@@ -114,12 +114,12 @@ if (typeof window === 'undefined') {
 }
 
 if (window.soundManager) {
-	soundManager.setup({url: 'https://play.pokemonshowdown.com/swf/'});
+	soundManager.setup({url: `https://${Config.routes.client}/swf/`});
 	if (window.Replays) soundManager.onready(window.Replays.soundReady);
 	soundManager.onready(() => {
 		soundManager.createSound({
 			id: 'notif',
-			url: 'https://play.pokemonshowdown.com/audio/notification.wav',
+			url: `https://${Config.routes.client}/audio/notification.wav`,
 		});
 	});
 }
```

Comment:
Wow, this is still here? It should probably be moved to `BattleScene`...

Reply:
OK, I'll do that.